recipes/antlr-cpp-runtime/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "antlr-cpp-runtime" %}
+{% set version = "4.12.0" %}
+{% set sha256 = "642d59854ddc0cebb5b23b2233ad0a8723eef20e66ef78b5b898d0a67556893b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```

Comment:
doesn't really do much... and _could_ theoretically confuse some tooling that does filename based stuff

Reply:
ah good point, let me remove. idk what this is used for anyway